### PR TITLE
Added rank support for RLBots

### PR DESCRIPTION
**Pull request by jeroen11dijk**
This should work but dont blame me when it doesnt.  Let me know when something doesn't work or should be done differently.

### CODE REVIEW

File: backend/utils/psyonix_api_handler.py [hunk 1]
```diff
@@ -137,6 +140,28 @@ def get_rank(steam_id):
     :param steam_id: steamid to get
     :return: rank, if it exists
     """
+    if len(steam_id) == 11 and steam_id[0] == 'b' and steam_id[-1] == 'b':
```
Comment (dtracers) [inline backend/utils/psyonix_api_handler.py]: Don't check length.

We also need to convert psyonix bots to be considered as all star.
I think we have an is bot test that you might be able to look at

File: backend/utils/psyonix_api_handler.py [hunk 2]
```diff
@@ -137,6 +140,28 @@ def get_rank(steam_id):
...
+        engine, Session = startup()
+        bot = Session().query(Player).filter(Player.platformid == steam_id).first().platformname
```
Comment (dtracers) [inline backend/utils/psyonix_api_handler.py]: Also might be good to move all of this to it's own method of possible.

File: backend/utils/psyonix_api_handler.py (hunk 1 above)
Comment (jeroen11dijk) [reply backend/utils/psyonix_api_handler.py]: I took that from sciguy's commit where he added support for bot stats.
(https://github.com/SaltieRL/DistributedReplays/blob/6e0aed4ae0e26079e7a774f58ab56495e55f3eb1/backend/blueprints/spa_api/service_layers/player/play_style.py#L70). But I will look into a better is bot method

Comment (dtracers) [reply backend/utils/psyonix_api_handler.py]: All star id is not 11 it's 4

**Pull request merged**